demo/src/main/java/com/restaurant/demo/controller/BillController.java

```diff
@@ -0,0 +1,38 @@
+package com.restaurant.demo.controller;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.restaurant.demo.entity.Bill;
+import com.restaurant.demo.service.BillService;
+
+@RestController
+@RequestMapping("/bills")
+public class BillController {
+	
+	@Autowired
+	private BillService billService;
+	
+	@GetMapping()
+	public List<Bill> listBills(){
+		return billService.listBills();
+	}
+	
+	@GetMapping("/user")
+	public List<Bill> listBillsByUser(@RequestParam(required = true) int id){
```

Comment:
this functionality must go in the user controller because it should be:
if you want to get all the bills of a specific user
-  **_/users/<user_id>/bills_**

if you want to get one specific bill of a specific user (additionally we should validate that this bill really belongs to that user, let's return an error message in case it doesn't)
-  **_/users/<user_id>/bills/<bill_id>_**